data/timesketch.conf

```diff
@@ -29,6 +29,12 @@ SQLALCHEMY_DATABASE_URI = 'postgresql://<USERNAME>:<PASSWORD>@localhost/timesket
 ELASTIC_HOST = '127.0.0.1'
 ELASTIC_PORT = 9200
 
+# Define what labels should be defined that make it so that a sketch and
+# timelines will not be deleted. This can be used to add a list of different
+# labels that ensure that a sketch and it's associated timelines cannot be
+# deleted.
+LABELS_TO_PREVENT_DELETION = ['lithold']
```

Comment:
maybe call the default example label something else? 'preserve', or 'protected'
WDYT?

Reply:
sure... we can do that.... maybe both?

Reply:
Yeah, lets do both

Reply:
that was already done